chapters/ch50.md

```diff
@@ -24,7 +24,7 @@ Ngược lại, xét một hướng tiếp cận hoàn toàn đầu-cuối:
 
 > To train this system, we would need a large dataset of (Image, Steering Direction) pairs. It is very time-consuming and expensive to have people drive cars around and record their steering direction to collect such data. You need a fleet of specially-instrumented cars, and a huge amount of driving to cover a wide range of possible scenarios. This makes an end-to-end system difficult to train. It is much easier to obtain a large dataset of labeled car or pedestrian images.
 
-Để huấn luyện hệ thống này, chúng ta sẽ cần một tập dữ liệu lớn các cặp (Ảnh, Hướng Bẻ Lái). Sẽ rất mất thời gian và tiền của để có người lái xe loanh quanh và ghi lại hướng bẻ lái để thu thập dữ liệu này. Bạn cần những chiếc xe có thiết kế đặc biện, một lượng lớn dữ liệu lái xe để đảm bảo bao quát đầy đủ các tình huống khả dĩ. Việc này khiến hệ thống đầu-cuối rất khó để huấn luyện. Sẽ dễ hơn nhiều để đạt được một tập dữ liệu lớn với ảnh xe hơi và người đi bộ.
+Để huấn luyện hệ thống này, chúng ta sẽ cần một tập dữ liệu lớn các cặp (Ảnh, Hướng Bẻ Lái). Sẽ rất mất thời gian và tiền của để có người lái xe loanh quanh và ghi lại hướng bẻ lái để thu thập dữ liệu này. Bạn cần những chiếc xe được gắng những thiết bị đặc biệt, một lượng lớn dữ liệu lái xe để đảm bảo bao quát đầy đủ các tình huống khả dĩ. Việc này khiến hệ thống đầu-cuối rất khó để huấn luyện. Sẽ dễ hơn nhiều để đạt được một tập dữ liệu lớn với ảnh xe hơi và người đi bộ.
```

Comment:
"được gắng những" --> "được gắn những"? 
 
"huge amount of driving" có thể dịch khác đi chút ko bác? ví dụ như "nhiều lần chạy xe" hoặc cái gì đó tương tự? vì cả câu gốc là mô tả quá trình lấy data (gắn thiết bị + chạy thử nhiều lần), cụm "một lượng lớn dữ liệu lái xe" về cơ bản nghĩa cũng không khác, nhưng nó là kết quả của quá trình lấy data.

Reply:
"huge amount of driving" có thể dịch là "một lượng lớn số lần lái xe"

Reply:
Mình đổi thành "một lượng lớn dữ liệu thu thập được khi lái xe". Hơi dài tí nhưng đủ ý. Vả lại đây không phải từ khóa nên cũng không yêu cầu quá ngắn gọn.